keyboards/ebastler/isometria_75/rev1/mcuconf.h

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
Just a heads up, you don't need this entire board files here, Basically, you jus need this: 
https://github.com/qmk/qmk_firmware/blob/01c7108db52677c2a4210bcda0afd2b1d3f7619f/platforms/chibios/keyboard-config-templates/mcuconf.h

Just add the undef/define for any changes you need to make.

Reply:
Does that already work for F072 MCUs? I thought only F303s were supported by the minimal configs so far. I changed it and it throws a ton of errors during compiling now :/